content/css/concepts/grids/terms/grid-template-areas/grid-template-areas.md

```diff
@@ -0,0 +1,122 @@
+---
+Title: 'grid-template-areas'
+Description: 'A property that defines a grid template by referencing the names of the grid areas.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Grid'
+  - 'Layout'
+  - 'Responsive'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+Defines a grid template by referencing the names of the `grid-area` property, establishing cells in the grid and assigning them names.
+The `grid-template` property can be used as a shorthand to define the areas as well as defining the sizes of the columns and rows.
```

Comment:
```suggestion
The `grid-template-areas` property defines a CSS grid by assigning a [`grid-area`](https://www.codecademy.com/resources/docs/css/grids/grid-area) name to each element in the template. The `grid-template` property can be used as a shorthand to define the areas as well as defining the sizes of the columns and rows.
```

I had a question about the last line:

> The `grid-template` property can be used as a shorthand to define the areas as well as defining the sizes of the columns and rows.

Doesn't this information pertain more to an entry about `grid-template` as opposed to one about `grid-template-areas`. (we actually seem to already have [an existing entry](https://www.codecademy.com/resources/docs/css/grids/grid-template) for `grid-template`.

Reply:
Maybe we could write something about how `grid-template-areas` is [collectively used with other properties](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template#constituent_properties) ([`grid-template-columns`](https://www.codecademy.com/resources/docs/css/grids/grid-template-columns) & [`grid-template-rows`](https://www.codecademy.com/resources/docs/css/grids/grid-template-rows)) under the `grid-template` shorthand property?

Reply:
> Maybe we could write something about how `grid-template-areas` is [collectively used with other properties](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template#constituent_properties) ([`grid-template-columns`](https://www.codecademy.com/resources/docs/css/grids/grid-template-columns) & [`grid-template-rows`](https://www.codecademy.com/resources/docs/css/grids/grid-template-rows)) under the `grid-template` shorthand property?

Should I write it on a new example, maybe below the syntax screenshot? So that I can provide another visual explanation of that relationship and how it works.

Reply:
We would just write a sentence here such as the following:

```
This property is collectively used with [`grid-template-columns`](https://www.codecademy.com/resources/docs/css/grids/grid-template-columns) and [`grid-template-rows`](https://www.codecademy.com/resources/docs/css/grids/grid-template-rows) under the [`grid-template`](https://www.codecademy.com/resources/docs/css/grids/grid-template) shorthand property.
```

Reply:
Ok thanks! I was about to over explore it. 🫢